--- ./Net_DIME-0.2/test/dime_message_test.php.org	Mon May 13 08:27:31 2002
+++ ./Net_DIME-0.2/test/dime_message_test.php	Mon Jul  8 01:11:00 2002
@@ -548,7 +548,7 @@
 $debug = FALSE;
 $data = NULL;
 $f = fopen('/tmp/dime.data','wb+');
-$dime = new DIME_Message($f,4096,$debug);
+$dime = new Net_DIME_Message($f,4096,$debug);
 $dime->sendData($test);
 #$dime->sendData($test);
 $dime->sendEndMessage();
@@ -556,7 +556,7 @@
 print "filesize: $size, datasize: ".strlen($test)."\n";
 rewind($f);
 
-$dime = new DIME_Message($f,4096,$debug);
+$dime = new Net_DIME_Message($f,4096,$debug);
 $err = $dime->read();
 if (PEAR::isError($err)) {
     print $err->getMessage()."\n";
@@ -568,4 +568,4 @@
     }
 }
 
-?>
\ No newline at end of file
+?>
--- ./Net_DIME-0.2/test/dime_record_test.php.org	Mon May 13 05:41:57 2002
+++ ./Net_DIME-0.2/test/dime_record_test.php	Mon Jul  8 01:11:42 2002
@@ -38,7 +38,7 @@
 ';
 
 $data = NULL;
-$dime = new DIME_Record($data);
+$dime = new Net_DIME_Record($data);
 #$dime->setMB();
 $dime->generateID();
 $dime->setType('text/plain');
@@ -49,4 +49,5 @@
 print chunk_split(bin2hex(substr($enc,8)),72)."\n";
 $dime->decode($enc);
 print_r($dime->_record);
-?>
\ No newline at end of file
+?>
+
--- ./Net_DIME-0.2/DIME.php.org	Sun Jul  7 18:25:41 2002
+++ ./Net_DIME-0.2/DIME.php	Mon Jul  8 01:12:20 2002
@@ -49,7 +49,7 @@
 
 define('DIME_RECORD_HEADER',8);
 
-class DIME_Record extends PEAR
+class Net_DIME_Record extends PEAR
 {
     // these are used to hold the padded length
     var $ID_LENGTH = 0;
@@ -71,7 +71,7 @@
      */
     var $_record = array(0,0x8000,0,'','','');
     
-    function DIME_Record($debug = FALSE)
+    function Net_DIME_Record($debug = FALSE)
     {
         $this->debug = $debug;
         if ($debug) $this->padstr = '*';
@@ -298,7 +298,7 @@
 }
 
 
-class DIME_Message extends PEAR
+class Net_DIME_Message extends PEAR
 {
 
     var $record_size = 4096;
@@ -323,7 +323,7 @@
      *
      * TODO: integrate with the php streams stuff
      */
-    function DIME_Message($stream=NULL, $record_size = 4096, $debug = FALSE)
+    function Net_DIME_Message($stream=NULL, $record_size = 4096, $debug = FALSE)
     {
         $this->stream = $stream;
         $this->record_size = $record_size;
@@ -332,7 +332,7 @@
     
     function _makeRecord(&$data, $typestr='', $id=NULL, $type=DIME_TYPE_UNKNOWN)
     {
-        $record = new DIME_Record($this->debug);
+        $record = new Net_DIME_Record($this->debug);
         if ($this->mb) {
             $record->setMB();
             // all subsequent records are not message begin!
@@ -503,14 +503,14 @@
     /**
      * _processData
      *
-     * creates DIME_Records from provided data
+     * creates Net_DIME_Records from provided data
      *
      */
     function _processData(&$data)
     {
         $leftover = NULL;
         if (!$this->_currentRecord) {
-            $this->_currentRecord = new DIME_Record($this->debug);
+            $this->_currentRecord = new Net_DIME_Record($this->debug);
             $data = $this->_currentRecord->decode($data);
         } else {
             $data = $this->_currentRecord->addData($data);
--- ./package.xml.org	Sun Jul  7 20:15:02 2002
+++ ./package.xml	Mon Jul  8 01:08:32 2002
@@ -23,8 +23,9 @@
 More testing needs to be done, but it encodes/decodes it's own messages.</notes>
     <filelist>
       <file role="php" baseinstalldir="Net" md5sum="ed812fd024025fed1071c8cf77ee43c1" name="DIME.php"/>
-      <file role="test" baseinstalldir="Net" md5sum="532cfa437638d3af49a9209516ddfa3b" name="test\dime_message_test.php"/>
-      <file role="test" baseinstalldir="Net" md5sum="c2107e8cf6b6e1fb478754f398529ed3" name="test\dime_record_test.php"/>
+      <file role="test" baseinstalldir="Net" md5sum="532cfa437638d3af49a9209516ddfa3b" name="test/dime_message_test.php"/>
+      <file role="test" baseinstalldir="Net" md5sum="c2107e8cf6b6e1fb478754f398529ed3" name="test/dime_record_test.php"/>
     </filelist>
   </release>
 </package>
+
